### PR TITLE
fixing RW-17

### PR DESCRIPTION
**Pull request by dabitdev**

### CODE REVIEW

File: brd-android/fabriik-common/src/main/res/values/text_styles.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
     <style name="TextStyle.Overline" parent="TextStyle.Body3"/>
 
     <style name="TextStyle.BottomTab">
-        <item name="android:textColor">@color/navigation_menu_item_tint</item>
+        <item name="android:textColor">@color/fabriik_shuttle_grey</item>
```
Comment (roknovak) [inline brd-android/fabriik-common/src/main/res/values/text_styles.xml:87]: `fabriik_shuttle_grey` is color from Fabriik app and it should not be used in RW app. Please, check Figma again

File: brd-android/app/src/main/res/layout/controller_home.xml [hunk 2]
```diff
@@ -179,7 +179,7 @@
                         android:layout_height="wrap_content"
                         android:layout_gravity="center"
                         android:src="@drawable/ic_menu_trade"
-                        app:tint="@color/ic_navigation_tab" />
+                        app:tint="@color/light_icons_03" />
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_home.xml:182]: Incorrect color, please check Figma again

File: brd-android/app/src/main/res/layout/controller_home.xml [hunk 3]
```diff
@@ -256,7 +256,7 @@
                         android:layout_height="wrap_content"
                         android:layout_gravity="center"
                         android:src="@drawable/ic_menu_more"
-                        app:tint="@color/ic_navigation_tab" />
+                        app:tint="@color/light_icons_03" />
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_home.xml:259]: Incorrect color, please check Figma again

File: brd-android/app/src/main/res/layout/controller_home.xml [hunk 4]
```diff
@@ -204,7 +204,7 @@
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
                         android:layout_gravity="center"
-                        android:src="@drawable/ic_buy_sell_selector" />
+                        android:src="@drawable/ic_buy_sell" />
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_home.xml:207]: This icon is using incorrect color, please check Figma again

**Review by majsavli — CHANGES_REQUESTED** [review]
Same comments as Rok, also change layout background color, update to coprrect colors of other elements

**Review by roknovak — APPROVED** [review]

**Pull request merged**